storm-core/src/jvm/backtype/storm/package-info.java

```diff
@@ -0,0 +1,7 @@
+/*
```

Comment:
1: this file will be generated in ./storm-core/target/build directory 
2:  add the package-info.java to source directory using the add-source goal